test.disabled/ejabberd_tests/tests/rest_client_SUITE.erl

```diff
@@ -439,12 +453,19 @@ get_room_messages(Client, RoomID, Count, Before) ->
     Path = erlang:iolist_to_binary(PathList),
     get_messages(Path, Creds).
 
-create_room({_AliceJID, _} = Creds, RoomID, Subject) ->
-    Room = #{name => RoomID,
+create_room({_AliceJID, _} = Creds, RoomName, Subject) ->
+    Room = #{name => RoomName,
              subject => Subject},
     {{<<"200">>, <<"OK">>}, {Result}} = rest_helper:post(<<"/rooms">>, Room, Creds),
     proplists:get_value(<<"id">>, Result).
 
+update_room({_AliceJID, _} = Creds, RoomName, Subject, RoomID) ->
```

Comment:
Sorry, wrong choice of word. I've meant action.

This function expects the call to return "201 Created", so naming it `update_room` is misleading.